### PR TITLE
Add music command

### PR DESCRIPTION
**Pull request by DRSchlaubi**

### CODE REVIEW

**Comment by warriorzz** [comment]
ok

File: src/main/kotlin/de/nycode/bankobot/commands/fun/BingBangCommand.kt [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+import kotlinx.coroutines.flow.take
+import kotlinx.coroutines.launch
+import kotlin.time.ExperimentalTime
+import kotlin.time.seconds
+
+@OptIn(ExperimentalTime::class)
+@ModuleName("Fun")
+@AutoWired
+fun bingBangCommand() = command("bingbang") {
+    description("lÄSST DICH SPAß HABEN")
```
Comment (warriorzz) [inline src/main/kotlin/de/nycode/bankobot/commands/fun/BingBangCommand.kt:53]: I guess this should be... upper case?

File: src/main/kotlin/de/nycode/bankobot/commands/fun/BingBangCommand.kt [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+        }
+
+        val guildId = guild?.id ?: return@invoke
+
+        val link = lavalink.getLink(guildId)
+        link.connectAudio(channelId)
+
+        val player = link.player
+
+        @Suppress("SpellCheckingInspection") // STFU THE NAME IS GREAT
```
Comment (warriorzz) [inline src/main/kotlin/de/nycode/bankobot/commands/fun/BingBangCommand.kt:73]: SeemsGood

**Review by warriorzz — COMMENTED** [review]
:)

**Review by NyCodeGHG — APPROVED** [review]
lgtm

**Pull request merged**